.circleci/Dockerfile

```diff
@@ -0,0 +1,64 @@
+FROM centos:7
+
+RUN yum update -y
+
+RUN  yum install -y \
+        https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm \
+        git \
+        yum-utils \
+        centos-release-scl
+
+RUN yum install -y \
+        devtoolset-8 \
+        java-1.8.0-openjdk \
+        java-1.8.0-openjdk-devel \
+        patch \
+        python36 \
```

Comment:
Out of curiosity and related to the last week memory problem - what is the exact version of python?

Reply:
It is the version with the memory leak :) I originally ran into a problem building tensorflow and I thought it might be due to python that I built (it turned out to be a problem with tensorflow and GCC7). I was using 3.6.7 that I built and I wanted to use python36 from centos repos to try something different.